Makefile

```diff
@@ -18,38 +12,15 @@ bootstrap:
 	@echo Nothing to do.
 
 docker-build:
+	curl -Ls https://dl.bintray.com/deis/deisci/objstorage-0314fc5-linux-amd64 -O
+	mv objstorage-0314fc5-linux-amd64 rootfs/bin/objstorage
+	chmod +x rootfs/bin/objstorage
 	docker build --rm -t ${IMAGE} rootfs
 	docker tag -f ${IMAGE} ${MUTABLE_IMAGE}
 
-deploy: docker-build docker-push kube-pod
-
-kube-pod: kube-service
-	kubectl create -f ${POD}
-
-kube-secrets:
-	- kubectl -s http://172.17.8.102:8080 create -f ${SEC}
-
-secrets:
-	perl -pi -e "s|access-key-id: .+|access-key-id: ${key}|g" ${SEC}
-	perl -pi -e "s|access-secret-key: .+|access-secret-key: ${secret}|g" ${SEC}
-	echo ${key} ${secret}
-
-kube-service: kube-secrets
-	- kubectl create -f ${SVC}
-	- kubectl create -f manifests/deis-minio-secretUser.yaml
-
-kube-clean:
-	- kubectl delete rc deis-${SHORT_NAME}-rc
-
-kube-mc:
-	kubectl create -f manifests/deis-mc-pod.yaml
```

Comment:
@kmala :100: on deleting these Kubernetes related build targets. But, can you remove them in a different PR?